### PR TITLE
chore: add CODEOWNERS

### PR DESCRIPTION
**Pull request by aarnphm**
Signed-off-by: Aaron Pham <29749331+aarnphm@users.noreply.github.com>

## What does this PR address?

Fixes #(issue)

## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, both `make format` and `make lint` script have passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [ ] Did your changes require updates to the documentation? Have you updated
  those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

## Who can help review?

Feel free to tag members/contributors who can help review your PR.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1 @@
+Atalaya Tech Inc.
```
Comment (parano) [inline AUTHORS:1]: remove this

File: MANIFEST.in [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 # Files to exclude in PyPI distribution
 exclude CONTRIBUTING.md GOVERNANCE.md CODE_OF_CONDUCT.md DEVELOPMENT.md SECURITY.md
-exclude Makefile MANIFEST.in
-exclude *.yml
+exclude Makefile MANIFEST.in AUTHORS
+exclude *.yml .mailmap
```
Comment (parano) [inline MANIFEST.in:6]: revert this change?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -45,7 +45,42 @@ jobs:
               - requirements/docs-requirements.txt
               - "docs/**"
 
-  codestyle_check:
+  codeowners-sanity:
```
Comment (parano) [inline .github/workflows/ci.yml:48]: we only need to run this when codeowner files have changed right?

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+/bentoml/_internal/cli/ @ssheng @parano @larme @bentoml/bentoml-reviewers
+/bentoml/_internal/yatai* @yetone @bentoml/bentoml-reviewers
+/bentoml/_internal/service/ @parano @bentoml/bentoml-reviewers
+/bentoml/_internal/utils/analytics @aarnphm @bentoml/bentoml-reviewers
+/bentoml/_internal/configuration/ @ssheng @bentoml/bentoml-reviewers
+/bentoml/_internal/models/ @sauyon @bentoml/bentoml-reviewers
+/bentoml/_internal/marshal/ @bojiang @bentoml/bentoml-reviewers
+/bentoml/_internal/*.py @sauyon @bentoml/bentoml-reviewers
+/bentoml/testing/ @bojiang @bentoml/bentoml-reviewers
+/docs/ @parano @ssheng
```
Comment (parano) [inline .github/CODEOWNERS:16]:
```suggestion
# User-facing API changes:
/bentoml/*.py @ssheng @parano 

# BentoML Internal
/bentoml/_internal/bento/docker/ @aarnphm 
/bentoml/_internal/bento/ @sauyon @parano 
/bentoml/_internal/server/ @bojiang @ssheng 
/bentoml/_internal/frameworks/ @larme 
/bentoml/_internal/io_descriptors/ @aarnphm @sauyon
/bentoml/_internal/runner/ @bojiang @sauyon
/bentoml/_internal/cli/ @aarnphm @larme
/bentoml/_internal/yatai* @yetone 
/bentoml/_internal/service/ @bojiang 
/bentoml/_internal/utils/analytics @ssheng @parano 
/bentoml/_internal/configuration/ @ssheng
/bentoml/_internal/models/ @sauyon
/bentoml/_internal/marshal/ @bojiang
/bentoml/_internal/testing/ @bojiang 
/bentoml/_internal/external_typing/ @sauyon 
/bentoml/_internal/*.py @sauyon

# Example code & notebooks
/examples/ @larme @ssheng 

# Release scripts
/scripts/ @ssheng 

# Github Configs
/.github/ @aarnphm @ssheng 

# Docs, Community and Governance
/docs/ @parano @ssheng
README.md @parano @ssheng 
GOVERNANCE.md @parano @ssheng 
CODE_OF_CONDUCT.md @ssheng @yubozhao 
CONTRIBUTING.md @yubozhao @ssheng 
DEVELOPMENT.md @sauyon @ssheng 

# External Typing
/typings/ @sauyon @ssheng
```

@ssheng

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,41 @@
+* @bentoml/bentoml-reviewers
```
Comment (parano) [inline .github/CODEOWNERS:1]: what does this line do?

Comment (aarnphm) [reply .github/CODEOWNERS:1]: oh let me remove that line.

Comment (aarnphm) [inline .github/CODEOWNERS:1]:
```suggestion
```

**Comment by aarnphm** [comment]
Discuss offline, closing for now.

File: MANIFEST.in [hunk 6]
```diff
@@ -13,6 +13,7 @@ prune typings
 prune scripts
 prune hooks
 prune docs
+prune examples
```
Comment (ssheng) [inline MANIFEST.in:16]: This should already be in the head of main.

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+/bentoml/_internal/cli/ @aarnphm @larme @bentoml/bentoml-reviewers
+/bentoml/_internal/yatai* @yetone @bentoml/bentoml-reviewers
+/bentoml/_internal/service/ @bojiang @bentoml/bentoml-reviewers
+/bentoml/_internal/utils/analytics @ssheng @parano  @bentoml/bentoml-reviewers
+/bentoml/_internal/configuration/ @ssheng @bentoml/bentoml-reviewers
+/bentoml/_internal/models/ @sauyon @bentoml/bentoml-reviewers
+/bentoml/_internal/marshal/ @bojiang @bentoml/bentoml-reviewers
+/bentoml/_internal/testing/ @bojiang @bentoml/bentoml-reviewers
+/bentoml/_internal/external_typing/ @sauyon @bentoml/bentoml-reviewers
+/bentoml/_internal/*.py @sauyon @bentoml/bentoml-reviewers
```
Comment (ssheng) [inline .github/CODEOWNERS:22]: Do you think this can be simplified to just the team, `@bentoml/bentoml-reviewers`?

**Review by ssheng — APPROVED** [review]
